testplan/testing/multitest/suite.py

```diff
@@ -447,17 +447,74 @@ def testcase(*args, **kwargs):
+                                        or list) if and only if there is a
+                                        single parametrization argument.
+    :param name_func: custom testcase name generation alogorithm.
+                        name_func(func_name, kwargs) => testcase_method_name
+                        Where:
+                        func_name - Name of the parametrization target function
+                        kwargs - The order of keys will be the same as the order
+                                of arguments in the original function
+    :type name_func: ``function(string, collections.OrderedDict) => str``
+    :param tag_func: dynamic testcase tag assignment function.
+                    tag_func(kwargs) => named_tag_context/ simple_tags
+                    Where:
+                        kwargs - parametrized keyword argument dictionary
+
+                    NOTE: If you use tag_func along with tags argument,
+                    testplan will merge the dynamically generated tag
+                    context with the explicitly passed tag values.
+    :type tag_func: ``tag_func(dict) => dict['str': str]/ list[str]``
+    :param docstring_func: custom testcase docstring generation function.
+                            docstring_func(docstring, kwargs) =>
+                            [parametrization_arguments]/
+                            [original_docstring, parametrization_arguments]
+                        Where:
+                        docstring - Name of the parametrization target function
+                        kwargs - The order of keys will be the same as the order
+                                of arguments in the original function
+    :type docstring_func: ``docstring_func(str/none, collections.OrderedDict) =>
+                            [dict[`any`: any]]/
+                            [str, dict[`any`: any]]``
+    :param custom_wrappers: wrapper to decorate paraametized testcases (
+                            used instead of @decorator syntax) that uses
+                            testplan.common.utils.callable.wraps()
+    :type custom_wrappers: ```custom_wrappers(func) => None``
+    :param summarize: Whether the testcase should be summarised in it's output
+    :type summarize: ``bool``
+    :param num_passing: The number of passing assertions reported per
+                        category per assertion type
+    :type num_passing: ``int`
+    :param num_failing:The number of failing assertions reported per
+                        category per assertion type
+    :type num_failing: ``int`
+    :param key_combs_limit: Max number of failed key combinations on fix/dict
+                            summaries.
+    :type key_combs_limit: ``int``
+    :param execution_group: group of test cases to run in parallel with,
+                            (groups overall are executed serially)
+    :type execution_group: ``str``
+    :param timeout: time elapsed until TimeoutException raised
```

Comment:
Can you mention that the value is given in seconds e.g. "time elapsed in seconds". It's a little annoying when timer-related functions don't make it clear if they take seconds, ms, ns etc.